src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle() {
+    }
+
+    public Circle(double radius) {
+        this.typeOfFigure = TypeOfFigure.CIRCLE;
+        this.area = getCircleArea(radius);
+        this.radius = radius;
+    }
+
+    public double getCircleArea(double radius) {
+        return Math.PI * Math.pow(radius,2);
+    }
```

Comment:
This method could be private since it's only used within this class. Also, it's better to use the class field 'radius' directly instead of passing it as a parameter.